### ISSUE TITLE
Service spamming mysql requests?

### ISSUE THREAD

**Issue by GeeZuZz** [traccar/traccar#4345]
I use the Windows service running in background, connecting to an external MySQL server.
I got rate blocked (presumably because of a different application).

Usually, I would get unblocked after 5 minutes. But the server admin said that I was constantly blocked again, because I was spamming requests (all failed "can't connect" because rate block). E.g, he saw 8 connection attempts in 5 seconds.

In other words, it seems that there's possibly a bug that causes too frequent connection retries when it can't connect to server or something?

Edit: It's possible I don't use the newest version, how do I check? (I only have service running)
